### ISSUE TITLE
Template application for racket templates

### ISSUE THREAD

**Issue by spdegabrielle** [racketscript/racketscript#276]
Would you be up for creating a simple template application for racket templates?

The idea is to make it easier for users to get started with RacketScript

Maybe the playground source would be appropriate ? 

https://github.com/racket-templates

**Comment by stchang** [comment]
So would it be appropriate to just copy the files from the playground?

**Comment by spdegabrielle** [comment]
Whatever you think would be best for getting someone new to RacketScript started?

is RacketScript playground written in RacketScript?

**Comment by spdegabrielle** [comment]
I'd like to get the template into a state where a user could click to deploy - is something like Glitch possible?

https://dev.to/glitch/adding-a-remix-on-glitch-button-to-your-github-project-515c

**Comment by stchang** [comment]
I've never used Glitch but iiuc it would be the compiled js that is deployed? So would that still be useful?

**Comment by stchang** [comment]
> is RacketScript playground written in RacketScript?

Yes, both client and server are 100% rs, but there are a lot of ffi calls to libraries like `express`

**Comment by stchang** [comment]
I ported the example node app from glitch.com: https://github.com/racketscript/glitch-template

It's set up so someone could clone and easily deploy by just typing `make`

You can see it running here: http://dev.racketscript.org:8080/

I've never used racket templates, but you can add this as an example there if you think it makes sense.
